Makefile

```diff
@@ -36,19 +46,33 @@ build: $(VIRTUALENV)
 upload: build
 	$(VIRTUALENV)/bin/twine upload dist/*
 
-test: $(VIRTUALENV)
+
+autoformat: $(VIRTUALENV)
+	$(VIRTUALENV)/bin/isort --recursive $(AUTOFORMAT_TARGETS)
+	if [ -f "$(VIRTUALENV)/bin/black" ]; then $(VIRTUALENV)/bin/black $(AUTOFORMAT_TARGETS); fi
+
+
+lint: $(VIRTUALENV)
```

Comment:
Why not just `lint: $(VIRTUALENV) autoformat` and get rid of the `black` and `isort` invocations here? Does `flake8` really need to go between the other steps? That doesn't bode well if so...